### PR TITLE
Fix some scaling issues

### LINKED ISSUE #541: Break panel icons do not resize
```
I'd rather open it up for comment to see if anyone else agrees first. 

One: If you're continually using them, you'd want something big you can easily click.
There's a bit of precision clicking going or a little extra care which is required
if you're clicking constantly at a high rate.

As a bonus, i've seen quite a few people starting off with zap miss them completely
because of their size and not know how to intercept. While yes, rtfm, ui that allows
you to easily infer how to use it is a good thing.

I do think burp got the right scale + having names on this one. What do people think
comparing the two?
```

Original issue reported on code.google.com by `contact@markcunningham.ie` on 2013-02-27 09:59:47

### PR DESCRIPTION
**Pull request by psiinon**
There will be others, but these are the ones I've spotted so far.
Fixes #541

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
So should this be "fix" for 541 or are you planning to make 541 more generic and leave it open?

**Review by kingthorin — APPROVED** [review]

**Comment by psiinon** [comment]
Part of 541 is a request to make the icons scalable independant of the font scaling.
This doesnt do that which is why I didnt mark it as fixing it, but open to suggestions ...

**Comment by kingthorin** [comment]
Let's break out another issue to scaling unrelated to Break buttons? Maybe a second one for separating options for image scaling vs text?

**Comment by psiinon** [comment]
I'll raise one for that..

**Comment by psiinon** [comment]
Doh! Only just realised getScaledSize(int) does what I need so no need for getScaledInteger(int) ;)

**Review by kingthorin — APPROVED** [review]

**Review by ricekot — APPROVED** [review]

File: zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanelToolbarFactory.java [hunk 1]
```diff
@@ -175,15 +176,17 @@ private void setActiveIcon(boolean active) {
             if (active) {
                 parent.setIconAt(
                         parent.indexOfComponent(breakPanel),
-                        new ImageIcon(
-                                BreakPanelToolbarFactory.class.getResource(
-                                        "/resource/icon/16/101.png"))); // Red X
+                        DisplayUtils.getScaledIcon(
+                                new ImageIcon(
+                                        BreakPanelToolbarFactory.class.getResource(
+                                                "/resource/icon/16/101.png")))); // Red X
```
Comment (thc202) [inline zap/src/main/java/org/zaproxy/zap/extension/brk/BreakPanelToolbarFactory.java:182]: Worth extracting a method (passing just the resource path).

**Comment by thc202** [comment]
Thank you!

**Pull request merged**